src/lib/Hydra/Helper/AddBuilds.pm

```diff
@@ -334,7 +352,8 @@ sub evalJobs {
 
     my $evaluator = ($exprType eq "guile") ? "hydra-eval-guile-jobs" : "hydra-eval-jobs";
 
-    my @cmd = ($evaluator, $nixExprFullPath, "--gc-roots-dir", getGCRootsDir, "-j", 1, inputsToArgs($inputInfo, $exprType));
+    my @args = inputsToArgs($inputInfo, $exprType);
+    my @cmd = ($evaluator, $nixExprFullPath, "--gc-roots-dir", getGCRootsDir, "-j", 1, @args);
```

Comment:
Ah, I used to pass a list of temporary file allocations so had to split this code, you're right, it is a redundant change after I stopped doing that.